### PR TITLE
ASIL field as enum, fix LAST-UPDATE, add PL field

### PR DESCRIPTION
**Pull request by ryushima**
I have imlemented the features. Thank you for your adivices.
I have done tested the script, please merge it.

### CODE REVIEW

File: FuSa-addon/scripts/REQIF_export.groovy [hunk 1]
```diff
@@ -51,33 +44,46 @@ def traverseTree(thisNode, parent_identifier)
...
-        XML.'REQ-IF-HEADER'('IDENTIFIER':'_adkamo'){
-            XML.'CREATION-TIME'(DATE)
-            XML.'REPOSITORY-ID'('?')
-            XML.'REQ-IF-TOOL-ID'('Freemind FuSa Extension v0.5')
-            XML.'REQ-IF-VERSION'('1.0')
-            XML.'SOURCE-TOOL-ID'('I guess this is only relevant for a round trip')
+        XML.'REQ-IF-HEADER'('IDENTIFIER':"_" + UUID.randomUUID().toString()){
+            XML.'CREATION-TIME'("${format.format(new Date())}")
+            XML.'REPOSITORY-ID'("__73f2bb4d-760e-44d9-99d3-5807d44b4928")
+            XML.'REQ-IF-TOOL-ID'("_5021db30-dd57-4612-bb75-98c4801619c0")
```
Comment (Jochen-Kall) [inline FuSa-addon/scripts/REQIF_export.groovy]: Is this change intentional? 
According to the REQIF spec, this field should identify the exporting tool to allow the importing side to see which tool/version generated the file, so I'd expect something human readable here.
Maybe it would be best to have a central tool and version identifier string for the addon to avoid having to maintain it here

File: FuSa-addon/scripts/REQIF_export.groovy [hunk 2]
```diff
@@ -51,33 +44,46 @@ def traverseTree(thisNode, parent_identifier)
...
+            XML.'REQ-IF-VERSION'('1.1')
```
Comment (Jochen-Kall) [inline FuSa-addon/scripts/REQIF_export.groovy]: According to the spec (v.1.2) this field always needs to contain "1.0"
That might be an error though, considering it is Spec version 1.2 ...
https://www.omg.org/spec/ReqIF/1.2/PDF (page 27)

File: FuSa-addon/scripts/REQIF_export.groovy [hunk 3]
```diff
@@ -51,33 +44,46 @@ def traverseTree(thisNode, parent_identifier)
...
+            XML.'SOURCE-TOOL-ID'("_63165248-e740-4271-a5e1-03ddc648d5eb")
```
Comment (Jochen-Kall) [inline FuSa-addon/scripts/REQIF_export.groovy]: Same applies as above, but If I read the spec right, this field is for roundtrip exchange of requirements going back and forth between tools, so for a first time export, I think it should be empty, not sure though.

Comment (ryushima) [reply FuSa-addon/scripts/REQIF_export.groovy]: Lack of consideration.. sorry.
I will change header such as bellow.

XML.'REQ-IF-TOOL-ID'('Freemind FuSa Extension v0.5')
XML.'REQ-IF-VERSION'('1.0')
XML.'SOURCE-TOOL-ID'('Freeplane')


*Doors NG 7.0.2 exported header sample ( round trip mode )
Doors NG has reqif export function, so REQ-IF-TOOL-ID and SOURCE-TOOL-ID is same.

&lt;THE-HEADER&gt;
		&lt;REQ-IF-HEADER IDENTIFIER=&quot;_5fef5cd0-1913-4611-8324-dd3074d54e65&quot;&gt;
			&lt;CREATION-TIME&gt;2021-06-20T22:08:31.230Z&lt;/CREATION-TIME&gt;
			&lt;REPOSITORY-ID&gt;&lt;/REPOSITORY-ID&gt;
			&lt;REQ-IF-TOOL-ID&gt;IBM Engineering Requirements 7.0.2&lt;/REQ-IF-TOOL-ID&gt;
			&lt;REQ-IF-VERSION&gt;1.0&lt;/REQ-IF-VERSION&gt;
			&lt;SOURCE-TOOL-ID&gt;IBM Engineering Requirements 7.0.2&lt;/SOURCE-TOOL-ID&gt;
			&lt;TITLE&gt;IBM Engineering Requirements Sun Jun 20 22:08:31 EDT 2021&lt;/TITLE&gt;
		&lt;/REQ-IF-HEADER&gt;
&lt;/THE-HEADER&gt;

*Jama exported header (export only mode, no round trip.)
Jama  Connect doesn't have reqif export function itself.
To export a reqif file, an user has to use External tool called Data Exchange.
REQ-IF-TOOL-ID is  Data Exchange and SOURCE-TOOL-ID is Jama Connect.
        &lt;REQ-IF-HEADER IDENTIFIER=&quot;_7067c9b1-64e8-45da-9765-1af1dfae7185&quot;&gt;
            &lt;CREATION-TIME&gt;2021-06-04T18:53:30.991+09:00&lt;/CREATION-TIME&gt;
            &lt;REQ-IF-TOOL-ID&gt;Data Exchange 3.5.0&lt;/REQ-IF-TOOL-ID&gt;
            &lt;REQ-IF-VERSION&gt;1.0&lt;/REQ-IF-VERSION&gt;
            &lt;SOURCE-TOOL-ID&gt;Jama Connect&lt;/SOURCE-TOOL-ID&gt;
            &lt;TITLE&gt;APCM&lt;/TITLE&gt;
        &lt;/REQ-IF-HEADER&gt;

Comment (Jochen-Kall) [reply FuSa-addon/scripts/REQIF_export.groovy]: Hi Ryushima san,

thanks, looks good, still one more typo left though, it should be
`XML.'REQ-IF-TOOL-ID'('Freeplane FuSa Extension v0.5')`

**Comment by ryushima** [comment]
Hi @Jochen-Kall san
I fixed  and pushed. please check it
